crates/gpui/src/platform.rs

```diff
@@ -532,6 +532,12 @@ impl Default for WindowOptions {
 }
 
 /// The options that can be configured for a window's titlebar
+// TODO: This interface is lacking information required for other windowing systems,
```

Comment:
Please mark this with `todo!(linux)` instead :)

Reply:
And actually, I think this whole comment should be moved next to the other ones when the decoration manager is initialized